gunshot/data-processing-application/ProtocolBuffer.java

```diff
@@ -0,0 +1,111 @@
+import java.util.HashMap;
+import java.util.ArrayList;
+/**
+ * ProtocolBuffer Class
+ * 
+ * <p>ProtocolBuffer is a class for protocol buffer creation from datasets in a csv format
```

Comment:
I'm a little bit confused about what the point of this class is. We should be able to use a protocol buffer directly right? https://developers.google.com/protocol-buffers/docs/javatutorial